src/cli/commands/main.cr

```diff
@@ -0,0 +1,18 @@
+require "admiral"
+
+module Tanda::CLI
+  class CLI::Commmands
+    class Main < Admiral::Command
+      define_help description: "tanda_cli"
+
+      def run
+        puts help
+      end
+
+      register_command me : Me, description: "Get information about yourself"
```

Comment:
Oops I was messing around and forgot to change these back to `register_sub_command`